JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -0,0 +1,12 @@
+The **Popup** UI component is a pop-up window that overlays the current view and displays specified content.
```

Comment:
I want to try and make this more user-friendly - like we discuss in Talk. 

First of all, this is the first sentence and it's a "definition" - which is what we usually see in textbooks. So I wanted to try and avoid the explanation (especially since Popup is a pretty self-explanatory name). 

Another issue is "specified content". Phrases like that sound too technical, add passive voice, while we can be saying something like "you can display anything you want". But in this particular case, I'd rather leave this until the section that deals with the content. 

I also want to leave the "overlay" part out and come back to it later - in the "Show and Hide" section.

So here I'd just start with what people can do in the demo:

```suggestion
Click the Details button under an employee picture to see what a Popup widget looks like. In a Popup, note the title area with the close button. Try other ways to to hide the popup window - click outside of it or press ESC.
```

Reply:
Should it be Close button instead of close button?